Understanding Ownership/The Slice Type/String Slices/task.md

```diff
@@ -0,0 +1,112 @@
+## String Slices
+
+A _string slice_ is a reference to part of a `String`, and it looks like this:
+
+```rust
+    let s = String::from("hello world");
+
+    let hello = &s[0..5];
+    let world = &s[6..11];
+```
+
+This is similar to taking a reference to the whole `String` but with the extra `[0..5]` bit. Rather than a reference to the entire `String`, it’s a reference to a portion of the `String`.
+
+We can create slices using a range within brackets by specifying `[starting_index..ending_index]`, where `starting_index` is the first position in the slice and `ending_index` is one more than the last position in the slice. Internally, the slice data structure stores the starting position and the length of the slice, which corresponds to `ending_index` minus `starting_index`. So in the case of `let world = &s[6..11];`, `world` would be a slice that contains a pointer to the 7th byte (counting from 1) of `s` with a length value of 5.
```

Comment:
and `ending_index` is none more than the last position

Reply:
it's last position + 1

Reply:
I corrected as follows: "and `ending_index` is the last position in the slice + 1"